docs/zh/week03/03-3.md

```diff
@@ -1,10 +1,9 @@
 ---
 lang: zh
 lang-ref: ch.03-3
-title: Properties of natural signals
+title: 神经信号的特性
 authors: Ashwin Bhola, Nyutian Long, Linfeng Zhang, and Poornima Haridas
 date: 15 February 2020
-translator: Matt Tso
```

Comment:
Why did you remove the `translator:` field?

Reply:
I mis-understand our format that day. so sry

Reply:
I try to fix title thing directly with github edit button. Do you think that ok?